cpp/inc/bond/stream/output_buffer.h

```diff
@@ -19,18 +21,19 @@ struct VariableUnsignedUnchecked
 {
     BOOST_STATIC_ASSERT(N < 10);
 
-    static uint32_t Write(uint8_t* p, T value)
+    static uint32_t Write(char* p, T value)
```

Comment:
There was no measurable performance implication due to these changes when I tested.